app/products.py

```diff
@@ -0,0 +1,114 @@
+"""
+ Must have the environment variables set properly prior to running the script
```

Comment:
It is very nice you added the requirement here but most importantly add a description of what the script does.
Description of what it does, what it is for, example how to execute it.
In this case how to execute it is probably simple but in general that is what I like to add.